lib/services/livesync/android-livesync-service.ts

```diff
@@ -17,6 +17,6 @@ export class AndroidLiveSyncService extends PlatformLiveSyncServiceBase implemen
 		return service;
 	}
 
-	public prepareForLiveSync(device: Mobile.IDevice, data: IProjectDir): void { /* */ }
+	public prepareForLiveSync(device: Mobile.IDevice, data: IProjectDir): Promise<void> { return Promise.resolve(); }
```

Comment:
Marking the method as `async` would be enough, no need to return Promise.resolve()